app/controllers/moderations_controller.rb

```diff
@@ -18,6 +18,11 @@ def index
   def article
     authorize(User, :moderation_routes?)
     @moderatable = Article.find_by(slug: params[:slug])
+    @adjustments = TagAdjustment.where(article_id: @moderatable.id)
+    @removed_adjustments = @adjustments.filter { |a| a.adjustment_type == "removal" } if @adjustments
+    @added_adjustments = @adjustments.filter { |a| a.adjustment_type == "addition" } if @adjustments
+    @already_adjusted_names = @adjustments.map(&:tag_name).join(", ") if @adjustments
```

Comment:
don't think you actually need the ifs here:

```ruby
TagAdjustment.none.filter { |a| a.adjustment_type == "removal" }
```

if there are no @adjustments, Rails returns an emty relationship, and thus filter will just iterate on an empty list

Reply:
Ok I just pushed up the changes to remove the ifs